src/scss/member-invite-modal.scss

```diff
@@ -0,0 +1,71 @@
+@import 'variables';
+
+.member-invite-modal {
+  .ant-list-empty-text {
+    display: none;
+  }
+
+  .ant-input-group-addon > .ant-input-search-button {
+    box-shadow: none;
+    display: flex;
+    align-items: center;
+    height: 37px;
+    box-sizing: content-box;
+    border: 1px solid #1890ff;
+
+    &:not(:disabled) {
+      background-color: lighten(map-get($variant-colors, 'primary'), 58%);
+    }
+
+    &:disabled {
+      border: 1px solid hsl(0, 0%, 78%);
+    }
+  }
+
+  .ant-list-footer {
+    padding-bottom: 0;
+  }
+
+  .group-member-list {
+    max-height: 275px;
+    overflow: auto;
+    margin-top: 8px;
+  }
+
+  .input-title {
+    font-size: 16px;
+    font-weight: 600;
+    margin-bottom: 8px;
+    margin-left: 4px;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  }
+
+  .description {
+    margin-bottom: 0;
+    margin-top: 8px;
+    font-size: 15px;
+    color: hsl(0, 0%, 40%);
+  }
+
+  .input-icon {
+    color: darken(map-get($variant-colors, 'primary'), 10%);
+  }
+
+  .remove-all-btn {
+    width: 100%;
+    border: none;
+    box-shadow: none;
+    padding: 10px 15px;
+    height: unset;
+    font-size: 16px;
+    color: hsl(0, 0%, 30%);
+    border: 1px solid transparent;
+  }
+
+  .remove-all-btn:hover:not(:disabled),
+  .delete-member-btn:hover {
+    background-color: lighten(map-get($variant-colors, 'danger'), 45%);
```

Comment:
My opinion but I'm not too sure on if I like how the color for the Remove All button fades in on hover, I would prefer if it was just red and had a width of about 40%, but again, thats just my preference.